paths_cli/tests/commands/test_contents.py

```diff
@@ -36,6 +37,7 @@ def test_contents(tps_fixture):
             "Trajectories: 1 unnamed item",
             f"Snapshots: {2*len(init_conds[0])} unnamed items", ""
         ]
+        assert_click_success(results)
         assert results.exit_code == 0
```

Comment:
the exit code is tested in the `assert_click_succes`
```suggestion
        assert_click_success(results)
```

Reply:
Strange... this and the other related suggestion can't actually be committed. It guess because the change occurs outside the PR's diff? In this case it lets you make the suggestion, but doesn't let me commit it. Seems like a GitHub bug, but I don't have time to try to find the (probably existing) report on it. Anyway, accepting this changes by making them locally.